### PR TITLE
README with instructions how to install and use EnTK.

### PR DESCRIPTION
**Pull request by iparask**
This pull request shows how to install and run EnTK. Please tell me if the instructions are clear enough.

Thank you

### CODE REVIEW

**Comment by bspitzbart** [comment]
This is very helpful. 
The gsissh and myproxy installation needs clarification. Vivek's script runs fine, but I still cannot get a connection. Perhaps there is another step?
```
bspitzbart@ubuntu:~$ myproxy-logon -s myproxy.xsede.org -l bspitz
Enter MyProxy pass phrase:
Failed to receive credentials.
ERROR from myproxy-server:
PAM authentication failed: Permission denied
```

Also, can you add examples to 
```
export RADICAL_PILOT_DBURL=<mongodburl>
```
I'm not sure what the URL looks like for the docker instance

And there should be some test of the installation at the end like:
```
(rp) bspitzbart@ubuntu:~$ radical-pilot-version
0.50.8
```
and a pointer to
```
entk_env/share/radical.pilot/examples
```

**Comment by iparask** [comment]
I updated the instructions based on Brad's comments.

**Review by AymenFJA — APPROVED** [review]

File: EnTKdevel/README.md [hunk 1]
```diff
@@ -1,5 +1,117 @@
...
+```
+~$ docker run -d --name entk_queues -P rabbitmq:3
+~$ docker ps
+CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS                                                                                                 NAMES
+d334b23418f2        rabbitmq:3          "docker-entrypoint..."   3 months ago        Up 3 months         0.0.0.0:32775->4369/tcp, 0.0.0.0:32774->5671/tcp, 0.0.0.0:32773->5672/tcp, 0.0.0.0:32772->25672/tcp   entk_queues
+```
+Verify that there is a port that looks like `0.0.0.0:32772->25672/tcp`. The second part should be the same.
+
+After RabbitMQ is installed, we need to install gsissh and my-proxy. Instructions
+can be found [here](https://github.com/vivek-bala/docs/blob/master/misc/gsissh_setup_stampede_ubuntu_xenial.sh)
```
Comment (AymenFJA) [inline EnTKdevel/README.md:31]: I think this link needs to be updated, there are some commands that does not work.

Comment (iparask) [reply EnTKdevel/README.md:31]: Let me check with @vivek-bala to see with we can do. Although that is usually trial and error.

**Comment by bspitzbart** [comment]
I noted under issue #23 "MyProxy does not create credentials" that Vivek's
script works with Ubuntu 16.04 (if you uncomment everything that is
commented) but not 18.04 (even after changing the codename xenial -> bionic)

**Comment by vivek-bala** [comment]
Hey @iparask @bspitzbart,  these instructions have only been tested on 16.* and 14.*. Some of the gsissh/certificate packages are/were not available/released for 18.04 (last I had checked). That might be a reason why the current instructions don't work for 18.04. That was one of the reasons I had to go back to 16.04 as well.

Although currently, I do see something popped up for 18.04 (bionic) at https://downloads.globus.org/toolkit/gt6/stable/deb/dists/index.html.

**Comment by bspitzbart** [comment]
Are the examples here:
https://radical-cybertools.github.io/radical-pilot/quick_start.html
still valid?

I get:
(/home/bspitzbart/entk_env) bspitzbart@ubuntu: $ python 00_getting_started.py

2018-10-01 15:24:04,762: rp.session.ubuntu.bspitzbart.017805.0001: MainProcess                     : MainThread     : ERROR   : failed to fet profile for pilot.0000
Traceback (most recent call last):
  File "/home/bspitzbart/entk_env/lib/python2.7/site-packages/radical/pilot/utils/session.py", line 168, in fetch_profiles
    profiles = sandbox.list('*.prof')
  File "/home/bspitzbart/entk_env/lib/python2.7/site-packages/saga/namespace/directory.py", line 243, in list
    return self._adaptor.list (pattern, flags, ttype=ttype)
  File "/home/bspitzbart/entk_env/lib/python2.7/site-packages/saga/adaptors/cpi/decorators.py", line 57, in wrap_function
    return sync_function (self, *args, **kwargs)
  File "/home/bspitzbart/entk_env/lib/python2.7/site-packages/saga/adaptors/shell/shell_file.py", line 473, in list
    % (ret, out))
NoSuccess: failed to list(): (2)(/bin/ls: cannot access '*.prof': No such file or directory
) (/home/bspitzbart/entk_env/lib/python2.7/site-packages/saga/adaptors/shell/shell_file.py +473 (list)  :  % (ret, out)))

**Comment by iparask** [comment]
Hello @bspitzbart, can you check the instructions one last time? I think it has all the necessary information.

**Comment by bspitzbart** [comment]
Yes. Thank you for adding the myproxy comment, especially the extra tip to use the commented instructions.
Just a couple of typos:
"vitrualenv rp"
"Mlad provides you with a similat URL" (btw, you probably saw that mlab is being aquired by MongoDB. Hopefully not much will change)

**Review by bspitzbart — APPROVED** [review]

**Pull request merged**